config/jobs/image-pushing/k8s-staging-kubebuilder.yaml

```diff
@@ -0,0 +1,23 @@
+postsubmits:
+
+  kubernetes-sigs/kubebuilder:
+    - name: post-kubebuilder-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-api-machinery-kubebuilder
```

Comment:
@yashsingh74 from where did you find this value?

Reply:
I find this from here - https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kubebuilder/kubebuilder-presubmits.yaml#L20